packages/components/psammead-image/README.md

```diff
@@ -2,31 +2,75 @@
 
 ## Description
 
-The `psammead-image` component is a styled `img` element.
+The `psammead-image` component has two images available. The default is a styled `img` element that uses Styled Components.
+
+There is an alternative AMP Image which can be used. It is a React component returning an `amp-img` element.
 
 ## Installation
 
-## Props
```

Comment:
[This doesn't match the contributing guidelines](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md). We want to be consistent in README structure for ease of use/reliability for users, but also because it allows us to ensure that any anchored links to stuff like `<COMPONENT>/README.md#Props` will always work.